SDWebImage/Core/SDImageIOCoder.m

```diff
@@ -57,7 +57,7 @@ + (instancetype)sharedCoder {
 }
 
 #pragma mark - Bitmap PDF representation
-+ (UIImage *)createBitmapPDFWithData:(nonnull NSData *)data pageNumber:(NSUInteger)pageNumber targetSize:(CGSize)targetSize preserveAspectRatio:(BOOL)preserveAspectRatio {
++ (UIImage *)createBitmapPDFWithData:(nonnull NSData *)data pageNumber:(NSUInteger)pageNumber targetSize:(CGSize)targetSize scaleMode:(SDImageScaleMode)scaleMode {
```

Comment:
This effects https://github.com/SDWebImage/SDWebImagePDFCoder

Because that coder use this private API :)